src/decider/agent/DeciderAgent.php

```diff
@@ -585,10 +586,10 @@ private function getCopyrightsToDisableFalsePositivesClutter($uploadId, $clutter
       if ($deactivatedCopyright['is_copyright'] == "t") {
         $action = '';
         $hash = '';
```

Comment:
Hey @Hero2323 , I just went through the CopyrightDao code to update text. It looks like you need to provide the hash of original copyright here. It should prevent the strange behavior you are experiencing with the declutter section.